helpers/control.py

```diff
@@ -323,18 +324,39 @@ def __init__(self, vhost_auto=True):
         self.config = tempesta.Config(vhost_auto=vhost_auto)
         self.stats = tempesta.Stats()
         self.host = tf_cfg.cfg.get("Tempesta", "hostname")
-        self.stop_procedures = [self.stop_tempesta, self.remove_config]
         self.check_config = True
+        self.clickhouse = ClickHouseFinder()
+        self.clean_logs_on_stop = True
+        self.__stop_procedures = [self.stop_tempesta, self.remove_config]
+
+    def __wait_while_logger_start(self):
+        if "mmap" not in self.config.get_config():
+            return
+
+        self.clickhouse.tfw_logger_wait_until_ready()
+
+    @property
+    def stop_procedures(self):
+        if self.clean_logs_on_stop:
```

Comment:
Let's add `clean_logs` in default `__stop_procedures` and add this `if` to `clean_logs` method
```python3
    def clean_logs(self):
        if self.clean_logs_on_stop:
            self.clickhouse.tfw_log_file_remove()
            self.clickhouse.access_log_clear()
```
We don't use the adding to `__stop_procedures`. This way the code will be simpler.